targets/targets.json

```diff
@@ -1256,7 +1256,7 @@
                 "macro_name": "LPTICKER_DELAY_TICKS"
             },
             "lpticker_lptim_clock": {
-                "help": "Default value for LPTIM clock (lpticker_lptim == 1). Value is the dividing factor. Choose 1, 2 or 4",
+                "help": "Default value for LPTIM clock (lpticker_lptim == 1). Value is the dividing factor. Choose 1, 2, 4, 8, 16, 32, 64 or 128",
```

Comment:
You could  add a new line here:
`"constraint": "bla bla" ?`

Reply:
Was not aware of this key, very usefull, done and thanks for the tip

Reply:
It's a new key...

It's a joke... only "value" key is used in the json parsing, all other key values are/should be ignored...

Reply:
ah ah interesting also because lack of availability to add comments in json file is always a missing feature (especially in `mbed_app.json` file.

But good to know I can add 

```json
            "comment": "My comment"
```

Not sure it will pass on `mbed_app.json` BTW